src/npt/arch/aarch64.rs

```diff
@@ -60,25 +61,37 @@ bitflags::bitflags! {
 enum MemType {
     Device = 0,
     Normal = 1,
+    NormalNonCache = 2,
```

Comment:
What is `MemType` for?

Reply:
The `MemType` is used to represent different types of memory attributes for memory mappings in a vmm system.

Reply:
![image](https://github.com/user-attachments/assets/e7308191-75dd-4f8b-8dca-c12120bb9ccb)
it is just like the memattr in stage1 pte